### ISSUE TITLE
Request: Fake Readouts to Apps

### ISSUE THREAD

**Issue by fxsoap** [fusionjack/Adhell3#17]
There are a large number of apps that require certain permissions to work and not crash. 
Can we have a fake readout (false data) of permissions? (ie location is London, phone type=Note 3) etc?

**Comment by fxsoap** [comment]
@fusionjack I know this isn't the basis of the app and I have no clue about how the API would work with this. Can you give us an idea how complicate this would be?

**Comment by fusionjack** [comment]
i don't think it is possible with knox sdk to do this feature. at least i didn't see any API of such a function.

**Comment by fxsoap** [comment]
@fusionjack ok, thought i'd at least ask, it seems to come up with permissions when you mess with them :D

**Comment by fusionjack** [comment]
i always messing around with the code which could introduce bugs :)
I'll label this as enhancement and leave it open, just in case this is possible with Knox SDK (which I doubt it).
